src/mmw/js/src/draw/views.js

```diff
@@ -148,7 +167,41 @@ var PlaceMarkerView = Marionette.ItemView.extend({
     }
 });
 
-function clearLayer(layer) {
+function changeOutlineLayer(endpoint, tableId, model) {
+    var map = App.getLeafletMap();
+
+    if (endpoint && tableId != undefined) {
+        var ol = App.outlineLayer = new L.TileLayer(endpoint + '.png');
```

Comment:
This might be an artifact from an older version of the code, but it looks like something is still assigned to `App` here and on the next line and it doesn't get user anywhere.